lib/src/features/symbol_point_renderer.dart

```diff
@@ -20,6 +22,7 @@ class SymbolPointRenderer extends FeatureRenderer {
     Style style,
     TileLayer layer,
     TileFeature feature,
+    double rotation,
```

Comment:
We should consider whether rotation indicates rotation of the tile relative to the viewport, or rotation of the symbol relative to the tile.